### ISSUE TITLE
TypeError on clicking "Show Collection Only" button 

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2381]
```
Uncaught TypeError: Cannot read property 'name' of undefined
```

Error refers to `setParam` function in [`toggleCollectionOnly`](https://github.com/ManifoldScholar/manifold/blob/95c22dec9712772f380901f6b074ec3173bdf6b9/client/src/backend/containers/resource-collection/Resources.js#L148) method in `ResourceCollection.Resources` container.

This can be tested on Edge at [this route](https://edge.manifoldapp.org/backend/projects/resource-collection/02f2923e-239e-4c70-8034-1b5abe20f22e/resources).

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by zdavis** [comment]
Fixed in e8c661d3cccc7347edaac8a533f8301256d93043!
